modules/test-utils/src/scene-renderer.js

```diff
@@ -112,7 +122,28 @@ export default class SceneRenderer {
     return promise;
   }
 
+  _getNextSceneIndex(sceneIndex) {
+    let nextSceneIndex = sceneIndex;
+    while (nextSceneIndex < this.scenes.length) {
+      const scene = this.scenes[nextSceneIndex];
+      let skipScene = false;
+      if (scene.gpuVendorSkipList) {
+        skipScene = scene.gpuVendorSkipList.reduce((skip, vendor) => {
```

Comment:
I think this would be equivalent and more compact?
````
const info = getContextInfo(gl)
const skip = scene.ignoreGPUs.some(gpu => info.vendor.indexOf(gpu) >= 0);
```

Reply:
Addressed, thanks!